pkgs/by-name/te/termsnap/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "termsnap";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "tomcur";
+    repo = pname;
+    rev = "${pname}-v${version}";
```

Superfluous use of `pname` is frowned upon (for details see #277994):
```suggestion
    repo = "termsnap";
    rev = "termsnap-v${version}";
```